tests/rptest/tests/partition_state_api_test.py

```diff
@@ -129,3 +134,69 @@ def test_controller_partition_state(self):
             leader_state = leaders[0]["raft_state"]
             assert "followers" in leader_state.keys() and len(
                 leader_state["followers"]) == 4
+
+    @cluster(num_nodes=5)
+    def test_local_summary(self):
+        admin = Admin(self.redpanda)
+        n_topics = 100
+
+        def produce():
+            kafka_tools = KafkaCliTools(self.redpanda)
+            for t in self.topics:
+                kafka_tools.produce(t.name, 1, 1, acks=1)
+
+        def sumsum_eventually(**expected):
+            self.logger.debug(f"{expected=}")
+
+            tolerance = 10
```

Comment:
I think it helps to add a comment here as to why a tolerance is needed and they don't fully converge.